### PR TITLE
docs: release notes for v0.6.0

### PR DESCRIPTION
**Pull request by chappjc**
View: https://github.com/decred/dcrdex/blob/af340855c01049e9e5136ce2be00c203814d58ed/docs/release-notes/release-notes-0.6.0.md

### CODE REVIEW

File: docs/release-notes/release-notes-0.6.0.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+Always record the "seed" that is generated by the DEX client on initialization (when you set your password the first time in the browser window). This is a string of characters that is used to derive DEX account identities and initialize any native/built-in wallets you create in the DEX client. **You must have this seed to recover funds** that you receive in such wallets if your lose access to your DEX client's data folder.
+
+Although DCRDEX looks and feels like a regular exchange, the decentralized aspect brings an expanded role to the client. Please take the time to read and understand the following:
+
+- If using external wallets, never shut them down while dexc is running.  Always stop dexc before stopping your wallets.
+- If you have to restart dexc with active orders or swaps, you must **immediately login again** with your app password when dexc starts up.
+- There is an "inaction timeout" when it becomes your client's turn to broadcast a transaction, so be sure not to stop dexc or lose connectivity for too long or you risk having your active orders and swaps/matches revoked. If you do have to restart dexc, just remember to login as soon as you start it up again.
```
Comment (JoeGruffins) [inline docs/release-notes/release-notes-0.6.0.md:13]: Is this only "when it becomes your client's turn to broadcast a transaction"? I think it starts if the client misses a ping/pong?

File: docs/release-notes/release-notes-0.6.0.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+- Restyle the Wallets page.  The new style is also responsive. (<https://github.com/decred/dcrdex/pull/1700> <!-- related: <https://github.com/decred/dcrdex/commit/4df561131bea62e185c946879ed9bb77790aee5c>, <https://github.com/decred/dcrdex/commit/4e2100d6a9124114e24570fda079f02520e1eba3>, <https://github.com/decred/dcrdex/commit/a891a7793e5fcc125ea7b952d486e69159d4e2b5>, <https://github.com/decred/dcrdex/commit/aa55b045090c4e99d895abb33dbaf1870ef988d6>, <https://github.com/decred/dcrdex/commit/1b50969f631e69bad17eba0963f4cc55bc7c4abf>, <https://github.com/decred/dcrdex/commit/2421de9dc04ac8fea89fea0167d38281c5eccc11>, <https://github.com/decred/dcrdex/commit/a40f58e7c6438eb4ab8ff869a674edc239db6723>, <https://github.com/decred/dcrdex/commit/424d6d223ea73344000b6f76fb5175c99b0edcbf> -->
```
Comment (JoeGruffins) [inline docs/release-notes/release-notes-0.6.0.md:23]: These two missing the closing parenthesis on the right.
image

**Review by JoeGruffins — APPROVED** [review]

File: docs/release-notes/release-notes-0.6.0.md (hunk 1 above)
Comment (chappjc) [reply docs/release-notes/release-notes-0.6.0.md:13]: ping/pong as in the connection?  No, it doesn't matter.  Their booked orders do eventually become unbooked, but as it pertains to active swaps the time disconnected won't cause a problem if they get back before it's too late.

**Comment by chappjc** [comment]
Merged for https://github.com/decred/decred-binaries/pull/149/files

**Pull request merged**